app/apis/api_kubernetes.py

```diff
@@ -747,6 +763,145 @@ def create_or_update_drucker_on_kubernetes(
             body=v2_beta1_horizontal_pod_autoscaler
         )
     """
+    return service_name
+
+def switch_drucker_service_model_assignment(
+        application_id:int, service_id:int, model_id:int):
+    """
+    Switch model assignment.
+    :param application_id:
+    :param service_id:
+    :param model_id:
+    :return:
+    """
+    mobj = db.session.query(Model).filter(
+        Model.application_id==application_id,Model.model_id==model_id).one()
```

Comment:
I think exception `NoResultFound` should be handled or logged in case `none` is found.

Reply:
Exceptions are caught by `api.errorhandler`.
So we can handle and log them.

Reply:
I see. I still prefer explicit error handling at the right place because it is under our control and enable us to fix quickly with more specific log, instead of stack trace in default error handler.

If we decide to follow above practice, review of whole code base is necessary though. Maybe it should be a TODO.

Reply:
Cloud you create `Issue` about this?
Error handling is one of the most important things. I agree with your opinion!

At this time, this PR is not for refactoring. Thank you.

Reply:
https://github.com/drucker/drucker-dashboard/issues/31 created.